app/app/users/page.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+import { useRouter } from "next/navigation";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Spinner from "react-bootstrap/Spinner";
+import Table from "react-bootstrap/Table";
+import AppBreadCrumb from "@/components/AppBreadCrumb";
+import { useUsersInfinite } from "@/utils/users";
+import { useToken } from "@/utils/auth";
+import AlignedLabel from "@/components/AlignedLabel";
+import { FaUserPlus, FaCopy } from "react-icons/fa6";
+
+export default function Users() {
+  const token = useToken();
+  const router = useRouter();
+  const { users, isLoading, error } = useUsersInfinite(token);
+
+  if (error) {
+    console.error(error);
+  }
+
+  return (
+    <>
+      <AppBreadCrumb />
+      <Card>
+        <Card.Header className="fs-5 fw-bold">Users</Card.Header>
+        <Card.Body>
+          <div className="mb-3">
+            {!isLoading && (
+              <>
+                <Button className="me-2">
+                  <AlignedLabel>
+                    <FaUserPlus className="me-2" />
+                    <span>Add user</span>
+                  </AlignedLabel>
+                </Button>
+                <Button>
+                  <AlignedLabel>
+                    <FaCopy className="me-2" />
+                    <span>Copy user emails</span>
+                  </AlignedLabel>
```

Comment:
I know these are on your list of todos, and there are issues written for them, but i went down a mini side rabbit hole looking at extensions or settings for vscode to highlight TODOs 

and you have todos written elsewhere, but maybe you remove the todo after you write up an issue??

Reply:
sure, that sounds good. there are a handful that feel to vague for an issue just yet, so i was just planning to sweep through them periodically. but if that's feeling too messy i can keep them in a doc. i don't want the code to look/feel crufty!

Reply:
i like them being in the code! i know there is robust testing before releases, but having it in the code could help prevent broken buttons sneak in to main

Reply:
oh i see you're pointing out that i should put a todo here!

Reply:
well i didnt actually point it out, now that im reading what i wrote